addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -413,6 +413,16 @@ MockServer.include({
         var notifications = [[header, data]];
         this._widget.call('bus_service', 'trigger', 'notification', notifications);
     },
+    /**
+     * Simulate the 'get_suggested_recipients' Python method, than provide the
+     * suggested partner list
+     *
+     * @private
+     * @param {Object} args
+     */
+    _mockMailGetSuggestedRecipients: function(args) {
+        return {"7": [[8, "\"Joel Willis\" <joel.willis63@example.com>", "Customer Email"]]};
+    },
```

Comment:
mocked methods in mock server must not use hard-coded value. At worse, returns nothing. Better: it returns something coherent with `data` in the mock server.

At a glance of the python code, this is model specific... so likely there should be no default mock server method, and instead test itself should mock with hard-coded values (not great, but it's the best right now).